core/src/main/java/org/sonarsource/sonarlint/core/client/api/connected/ConnectedSonarLintEngine.java

```diff
@@ -71,7 +71,9 @@ public interface ConnectedSonarLintEngine extends SonarLintEngine {
    * @param projectBinding information about the project (must have been previously updated with {@link #updateProject(EndpointParams, HttpClient, String, ClientProgressMonitor)})
    * @param filePath       relative to the project.
    * @return All server issues in the local storage for the given file. If file has no issues, an empty list is returned.
+   * @deprecated tracking is managed by the new backend
    */
+  @Deprecated(since = "9.3")
```

Comment:
Is it not too early to deprecate? Do we already have full replacement of the matching/tracking for all type of issues?

Reply:
Your comment is about `getServerIssues()` and yes we have a full replacement for that, since the work on muting local-only issues. It's already not used anymore by SLI and SLLS.